chart/values.yaml

```diff
@@ -13,6 +13,8 @@ image:
   repo: openebs
   # -- Release tag for our images
   tag: develop
+  # -- Chart
+  chart: mayastor
```

Comment:
```suggestion
  prefix: mayastor
```

Do we want this to just be the prefix, or do we want this to be used for other things?

Reply:
done with the changes.